From 1e4f84b48dddf4f7526c17ceecf81377f9cbc490 Mon Sep 17 00:00:00 2001
From: Jonas Aaberg <cja@gmx.net>
Date: Sun, 26 Mar 2017 08:08:41 +0200
Subject: [PATCH 05/44] cht-vgpio: Add Asus T100HAN

Signed-off-by: Jonas Aaberg <cja@gmx.net>
---
 drivers/platform/x86/cht-vgpio.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/platform/x86/cht-vgpio.c b/drivers/platform/x86/cht-vgpio.c
index c0e4b187dbe1..59a2ae3ebba1 100644
--- a/drivers/platform/x86/cht-vgpio.c
+++ b/drivers/platform/x86/cht-vgpio.c
@@ -105,6 +105,13 @@ static struct dmi_system_id cht_vgpio_dmi_table[] __initdata = {
 			DMI_MATCH(DMI_PRODUCT_NAME, "E200HA"),
 		},
 	},
+	{
+		.ident = "ASUS T100HAN",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "ASUSTeK COMPUTER INC."),
+			DMI_MATCH(DMI_PRODUCT_NAME, "T100HAN"),
+		},
+	},
 	{}
 };
 MODULE_DEVICE_TABLE(dmi, cht_vgpio_dmi_table);
-- 
2.12.1

